Describing how to add product review from the storefront (#1217)
magento/merchdocs: src/customers/account-dashboard-my-product-reviews.md

    ![]({% link images/images-ee/account-dashboard-my-product-reviews-details.png %}){: .zoom}
    _Review Details_

## Add product review from the storefront

> When the **Magento Product Reviews** function is [enabled]({% link icatalog/catalog.md %}), customers can write reviews for any product in your catalog. Reviews can be written from the product page by clicking:

Author: Hi @erikmarr ☺️
Thank you so much for reviewing.
I have implemented this suggestion, could you please check and approve this?

Thank you!

> - Add **Your Review** for products with existing reviews.

Author: Done, please check!

Thanks!

> - Be the first to review this product for products with no existing reviews.

Author: Done, please check!

Thanks!

> 1. On the storefront open the **product page**.

Author: Done, please check!

Thanks!

> 1. Scroll down to the details block and select the **Reviews** tab.

Author: Done, please check!

Thanks!

> 1. Fill in required fields:

Reviewer: ```suggestion
1. Fills in required fields:
```

Author: Done, please check!

Thanks!

>    - Fill in the rating stars.

Author: Done, please check!

Thanks!

>    - Enter the **Nickname**.

Author: Done, please check!

Thanks!

>    - Describe the **Summary** of the review.

Author: Done, please check!

Thanks!

>    - Add a detailed description to the **Review** field.

Reviewer: ```suggestion
   - Detailed description in the **Review** field
```

Author: Done, please check!

Thanks!

> 1. Click <span class="btn">Save<span>.

Reviewer: ```suggestion
1. Clicks <span class="btn">Submit Review<span>.
```

Author: Done, please check!

Thanks!

{:.bs-callout-info}

> The submitted review will appear on the product page after the Magento admin approves it.

Reviewer: ```suggestion
The submitted review appears on the product page after the Magento admin approves it.
```

Author: Done, please check!

Thanks!
